Readme.md

```diff
@@ -0,0 +1,51 @@
+# BeCJK & History
+
+![screenshot](screenshot.png)
+
+BeCJK, is an input method for the legancy BeOS, first Developed by
+Anthony.Lee.
+
+Now, this copy of code is gotten from Andraw Bachmann.
+
+As time goes on, expecially with Haiku PM, BeCJK can't work normally on
+recently release of Haiku.
+
```

Comment:
this comment refers to BeCJK not working with Haiku now that there is package management.  You should check this and see if that is the case or not.  You may need to write a recipe to check this.

Reply:
I installed the app through HaikuDepot but even I can't figure out if the app is working properly because I don't know any of these(CJK) language. I changed language from the icons shown in the interface and tried to type something I think it is working normally. Cannot be sure about it.